modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -8,21 +8,26 @@ This procedure requires https://nodejs.org/[Node.js (and npm)].
 
 == Procedure
 
-. Create a Svelte application using the https://github.com/sveltejs/template[Svelte template project], for example:
+. Use the https://github.com/vitejs/vite[Vite] package to create a new Svelte project named `+tinymce-svelte-demo+`, such as:
 +
 [source,sh]
 ----
-npx degit sveltejs/template my-tiny-app
-cd my-tiny-app
+npm create vite@5 tinymce-svelte-demo -- --template svelte
```

Comment:
- Using `vite@5` here instead of `vite@latest` to ensure any major updates to Vite do not break these instructions.
- Using the Vite `svelte` template. Users can also use `svelte-ts` or [SvelteKit](https://kit.svelte.dev/docs/creating-a-project) here. Not sure if it's necessary to mention this.